src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -108,8 +112,26 @@ This resets the cluster to its initial state.`,
 			return nil
 		}),
 	}
+	var port int
+	portForward := &cobra.Command{
+		Use:   "port-forward",
+		Short: "Forward a port on the local machine to pachd. This command blocks.",
+		Long:  "Forward a port on the local machine to pachd. This command blocks.",
+		Run: pkgcobra.RunFixedArgs(0, func(args []string) error {
+			stdin := strings.NewReader(fmt.Sprintf(`
+pod=$(kubectl get pod -l app=pachd |  awk '{if (NR!=1) { print $1; exit 0 }}')
+kubectl port-forward "$pod" %d:650
+`, port))
+			return pkgexec.RunIO(pkgexec.IO{
+				Stdin:  stdin,
+				Stderr: os.Stderr,
+			}, "sh")
+		}),
+	}
```

Comment:
Would like to see some tests for this (e.g. what happens if kubectl DNE or cannot connect?), but thats out of scope for now. Lets add these cases to this issue so that we revisit it once we have the test harness: https://github.com/pachyderm/pachyderm/issues/710